src/scss/colors.scss

```diff
@@ -0,0 +1,25 @@
+//Main colors
+$game-background: #eee8ea;
+$accent-color: #591D2D;
+$accent-color-hover: #73263A;
+
+//Text colors
+$text: #4d4d4d;
+$text-light: #666666;
+$tile-text: #ffffff;
```

Comment:
What is the difference between `$title` and `$title-text`?
Doesn't make sense semantically